### PR TITLE
Proj4 overview & quickstart translation to fr

### PR DESCRIPTION
**Pull request by julien2512**
First translation try for proj4 overview & quickstart

### CODE REVIEW

**Comment by cvvergara** [comment]
Hello @julien2512 
I wonder if you have a travis account
https://travis-ci.org/
I want to know why the
https://travis-ci.org/OSGeo/OSGeoLive-doc/builds/247253941
is stalled
So, if you add a travis account and make it test on your fork, maybe it gets unstalled

File: fr/overview/proj4_overview.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  :align: right
+  :target: http://www.osgeo.org/incubator/process/principles.html
+
+Proj.4
+================================================================================
+
+Proj4 est une librairie qui fournit des outils pour transformer des points
+entre différents systèmes de coordonnées de référence. Un système de coordonnées géographique de référence permet de décrire tous les points de la Terre sous la forme d'un ensemble de coordonnées (comme la latitude, la longitude et l'élévation). Différents systèmes sont utilisés pour représenter les 3 dimensions de la Terre sur une carte plate de 2 dimensions.
+
+Proj4 est écrit en C++. Il est utilisé dans :doc:`MapServer <mapserver_overview>`, :doc:`GRASS GIS <grass_overview>`, :doc:`PostGIS <postgis_overview>`, :doc:`Mapnik <mapnik_overview>`, Thuban, OGDI, TopoCad, OGRCoordinateTransformation et bien d'autres.
```
Comment (rouault) [inline fr/overview/proj4_overview.rst]: C, pas C++

File: fr/overview/proj4_overview.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Proj.4
+================================================================================
+
+Proj4 est une librairie qui fournit des outils pour transformer des points
+entre différents systèmes de coordonnées de référence. Un système de coordonnées géographique de référence permet de décrire tous les points de la Terre sous la forme d'un ensemble de coordonnées (comme la latitude, la longitude et l'élévation). Différents systèmes sont utilisés pour représenter les 3 dimensions de la Terre sur une carte plate de 2 dimensions.
+
+Proj4 est écrit en C++. Il est utilisé dans :doc:`MapServer <mapserver_overview>`, :doc:`GRASS GIS <grass_overview>`, :doc:`PostGIS <postgis_overview>`, :doc:`Mapnik <mapnik_overview>`, Thuban, OGDI, TopoCad, OGRCoordinateTransformation et bien d'autres. 
+ 
+Proj4 a été traduit du C dans un grand nombre d'autres languages (Voir `The group of Proj4 projects (MetaCRS) <http://trac.osgeo.org/metacrs/wiki>`_). Certains sont utilisables via l'interface en ligne de commande.
+Ces sous-projects incluent
```
Comment (rouault) [inline fr/overview/proj4_overview.rst]: Quels sous-projets ? On dirait qu'on parle de sous-projets de proj.4 alors qu'il s'agit de sous-projets de MetaCRS dont certains non rien à voir avec proj.4

**Comment by julien2512** [comment]
Hi Vicky,

It seems the travis build has not started yet ! May be too much request at this time, I don't know.

Regards,
Julien.

File: fr/quickstart/proj4_quickstart.rst [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+================================================================================
+
+.. note:: 
+
+  **Que vais-je apprendre ?** Dans cette section, vous apprendrez à vous servir de l'outil proj. 
+
+Qu'est-ce que proj ?
+--------------------------------------------------------------------------------
+
+Proj and invproj perform respective forward  and inverse  transformation  of cartographic data to
```
Comment (rouault) [inline fr/quickstart/proj4_quickstart.rst]: Pas traduit

**Comment by julien2512** [comment]
The only weird thing in my PR is the symbolic links I scratched away in a first time indeed. May it break Travis ?!

**Comment by cvvergara** [comment]
@julien2512 
No, the symbolic links have nothing to do, they are not part of the repository they are created when the documentation is build locally

**Comment by julien2512** [comment]
@cvvergara The travis worker finally passed !

File: fr/overview/proj4_overview.rst (hunk 1 above)
Comment (julien2512) [inline fr/overview/proj4_overview.rst]: Yes

Comment (julien2512) [inline fr/overview/proj4_overview.rst]: done

File: fr/overview/proj4_overview.rst (hunk 2 above)
Comment (julien2512) [inline fr/overview/proj4_overview.rst]: Oui, d'accord, il y a encore des restes à corriger !
Yes, I agree, there's a bit more of MetaCRS to clean !

Comment (julien2512) [inline fr/overview/proj4_overview.rst]: done

File: fr/quickstart/proj4_quickstart.rst (hunk 3 above)
Comment (julien2512) [inline fr/quickstart/proj4_quickstart.rst]: I am a bilingual secretary now.

Comment (julien2512) [inline fr/quickstart/proj4_quickstart.rst]: done

**Comment by kalxas** [comment]
@cvvergara @rouault any other comments or should we merge?

**Comment by cvvergara** [comment]
@kalxas 
If you merge then make the change manually.

**Pull request merged**